Source/WebKit/NetworkProcess/mac/com.apple.WebKit.NetworkProcess.sb.in

```diff
@@ -879,3 +879,8 @@
 #if ENABLE(LLVM_PROFILE_GENERATION)
 (allow file-write* (subpath "/private/tmp/WebKitPGO"))
 #endif
+
+#if HAVE(WEBCONTENTRESTRICTIONS) && PLATFORM(MAC)
+(allow mach-lookup
+    (global-name "com.apple.familycontrols"))
```

Comment:
Perhaps we can conditionalize the access to this daemon on the feature being enabled? That does not need to land as part of this patch, and can be done as a follow-up, though.

Reply:
Yes, the sandbox rule would look something like this:

```
    (allow mach-lookup
        (require-all
            (extension "com.apple.webkit.extension.mach")
            (global-name "com.apple.familycontrols")))

```

Reply:
This can also be done in a follow-up :)